examples/nucleo_f429zi/nanopb/main.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2022, Petteri Aimonen <jpa at nanopb.mail.kapsi.fi>, Lucas Moesch
+ */
```

Comment:
MPLv2?

Reply:
zlib License stated in the repo - never heard of, but stated as campatible with GPL